platforms/org.eclipse.mita.platform.xdk110/src/org/eclipse/mita/platform/xdk110/connectivity/BleValidator.xtend

```diff
@@ -46,4 +48,22 @@ class BleValidator implements IResourceValidator {
 		}
 	}
 
+    protected def validateServiceType(SystemResourceSetup setup, ValidationMessageAcceptor acceptor) {
+
+		val ServiceConfigItem = setup.configurationItemValues.findFirst[x|x.item.name == 'Service'];
+		val CustomService = StaticValueInferrer.infer(ServiceConfigItem.value, []);
+		if (CustomService instanceof Enumerator) {
+			if (CustomService.name == "BLE_USER_CUSTOM_SERVICE") {
+				val customcharacteristic = setup.configurationItemValues.findFirst[it.item.name == "serviceUID"];
```

Comment:
Please use camelCaseForVariables consisting of multiple words.

Reply:
changed to camelcase